test/test_jit.py

```diff
@@ -11209,8 +11209,8 @@ def test_remove_dropout(self):
         class M(torch.nn.Module):
             def __init__(self):
                 super(M, self).__init__()
-                self.weight_0 = torch.nn.Parameter(torch.Tensor(torch.rand(weight_0_shape)))
-                self.weight_1 = torch.nn.Parameter(torch.Tensor(torch.rand(weight_1_shape)))
+                self.weight_0 = torch.nn.Parameter(torch.rand(weight_0_shape))
```

Comment:
There are lots of things similar to this one:

```python
... = torch.nn.Parameter(torch.Tensor(torch.rand(weight_0_shape)))
```

It seems redundant to me, so I removed the `Tensor` call. Is there a reason for wrapping `rand` into a `Tensor`?

Reply:
There's no reason, that just looks very weird. Looks like a good cleanup to me.